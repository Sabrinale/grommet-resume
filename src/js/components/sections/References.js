import React from 'react'

import { Description } from '../Description'
import { Highlights } from '../Highlights'
import { SectionTitle } from '../SectionTitle'

const Reference = ({ item }) => (
  <Highlights>
    <SectionTitle>{item.name}</SectionTitle>
    <Description>{item.reference}</Description>
  </Highlights>
)

export { Reference }
