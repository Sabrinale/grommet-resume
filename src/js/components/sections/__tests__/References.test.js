import React from 'react'
import renderer from 'react-test-renderer'

import { Reference } from '../References'

import defaultResume from './mockResume.json'

test('renders references section', () => {
  const data = defaultResume.references[0]
  const component = renderer.create(<Reference item={data} />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
