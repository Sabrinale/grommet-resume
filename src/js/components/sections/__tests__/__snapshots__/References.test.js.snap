// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders references section 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 kFdcfG"
>
  <h3
    className="StyledHeading-sc-1rdh4aw-0 jVEgcZ"
  >
    Jane Doe
  </h3>
  <p
    className="StyledParagraph-tbetod-0 jzEULC"
  >
    Reference...
  </p>
</div>
`;
